storage/migrations/07_runtime_accounts_stats.up.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+DROP INDEX IF EXISTS chain.ix_runtime_transfers_sender;
+DROP INDEX IF EXISTS chain.ix_runtime_transfers_receiver;
+
+CREATE INDEX ix_runtime_transfers_sender ON chain.runtime_transfers(runtime, sender);
```

Comment:
If we are adding the columns to runtime accounts are these indices needed?

Do the `*Recompute` methods end up using these indices?

Reply:
Good catch - I thought the recompute queries would use the indices but turns out they don't. Removed them entirely.